phpseclib/Crypt/Rijndael.php

```diff
@@ -755,7 +755,7 @@ function _subWord($word)
     {
         static $sbox;
         if (empty($sbox)) {
-            list(,,,, $sbox) = $this->_getTables();
+            list(, , , , $sbox) = $this->_getTables();
```

Comment:
It's listed in their upgrading guide btw: http://php.net/manual/en/migration70.incompatible.php, under "Empty list() assignments have been removed".